doc/editor_setup.mld

```diff
@@ -0,0 +1,92 @@
+{0 OCamlFormat Manual}
+
+{1 Editor setup}
+
+{2 Disable outside project}
+
+As mentioned in the Options section, when the option `--disable-outside-detected-project` is set, .ocamlformat files outside of the project (including the one in `XDG_CONFIG_HOME`) are not read. The project root of an input file is taken to be the nearest ancestor directory that contains a .git or .hg or dune-project file. If no config file is found, then the formatting is disabled.
+
+This feature is often the behavior you can expect from OCamlFormat when it is directly run from your text editor, so it is advised to use this option.
```

Comment:
I'd start this section with "Make sure you have a `.ocamlformat` file at the root of your project". We frequently have issues in VSCode for this, so it's good to surface the need to create this file in this Editor Setup doc of ocamlformat 🙂

Reply:
done